Packs/CVESearch/Integrations/CVESearch/CVESearch.py

```diff
@@ -0,0 +1,127 @@
+import urllib3
+from typing import Dict, Any, Tuple, List, Union
+
+from CommonServerPython import *
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+
+    def __init__(self, base_url: str):
+        header = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+        super().__init__(base_url=base_url, headers=header)
+
+    def http_request(self, suffix: str) -> Union[Dict[str, Any], List[Dict[str, Any]]]:
+
+        """Connects to api and Returns response.
+           Args:
+               suffix :The API endpoint.
+           Returns:
+               response from the api.
+           """
+        return self._http_request(method="GET", url_suffix=suffix)
+
+
+def cve_to_context(cve):
+    return {
+        "ID": cve.get("id", ""),
+        "CVSS": cve.get("cvss", "0"),
+        "Published": cve.get("Published", "").rstrip("Z"),
+        "Modified": cve.get("Modified", "").rstrip("Z"),
+        "Description": cve.get("summary", '')
+    }
+
+
+def test_module(client: Client, *_):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    res = client.http_request(suffix="/last")
+    if res:
+        return "ok", None, None
+    raise Exception('Error occurred while trying to query the api.')
+
+
+def cve_search(client: Client, args: dict) -> Tuple[Any, Dict[str, List[Any]], List[Union[Dict[str, Any], List[Dict[str, Any]]]]]:
+    cve_ids = args.get("cve_id")
+    if not cve_ids:
+        raise KeyError("cve_id argument not given")
+    cve_ids = argToList(cve_ids)
+    raw_response = []
+    ec = []
+    human_readable = []
+    cve_ids_with_no_response = []
+    for cve_id in cve_ids:
+        res = client.http_request("cve/" + cve_id)
+        if not res:
+            # If there is no data in CVE-Search for a given cve-id the response will return None,
+            # in this case these ids are stored in order to let the client know that querying those ids gave no results.
+            cve_ids_with_no_response.append({"ID": cve_id})
+            continue
+        data = cve_to_context(res)
```

Comment:
Nice idea of outputing the ids with no data on line 70, but if `res` is `none` we will append each iteration a dict that  looks like this:
```
   {
        "ID": "",
        "CVSS": "0",
        "Published": "",
        "Modified": "",
        "Description": ""
    }
```
Is that something we want?

Reply:
It should not happen because of the ```continue```, it should skip lines 70-73.